landing-strip/core/src/main/java/com/novoda/landingstrip/FixedWidthTabsContainer.java

```diff
@@ -0,0 +1,33 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+public class FixedWidthTabsContainer extends TabsContainer {
+
+    static TabsContainer newInstance(Context context, Attributes attributes, AttributeSet attrs) {
+        LinearLayout tabsContainerView = new LinearLayout(context);
+        tabsContainerView.setOrientation(LinearLayout.HORIZONTAL);
+        ViewGroup.LayoutParams layoutParams = tabsContainerView.generateLayoutParams(attrs);
+        layoutParams.width = ViewGroup.LayoutParams.MATCH_PARENT;
+        tabsContainerView.setLayoutParams(layoutParams);
+        tabsContainerView.setPadding(attributes.getTabsPaddingLeft(), 0, attributes.getTabsPaddingRight(), 0);
+
+        return new FixedWidthTabsContainer(tabsContainerView);
+    }
+
+    FixedWidthTabsContainer(LinearLayout tabsContainerView) {
+        super(tabsContainerView);
+    }
+
+    @Override
+    void addTab(View tabView, int position, CharSequence title) {
+        super.addTab(tabView, position, title);
+        LinearLayout.LayoutParams layoutParams = (LinearLayout.LayoutParams) tabView.getLayoutParams();
+        layoutParams.weight = 1;
```

Comment:
this weight should be provided by the tab, no need to force a 1! (unless it's not being picked up)